applications/API/models.py

```
@@ -0,0 +1,626 @@
+from django.db import IntegrityError, models
+from django.db.models.signals import post_delete, post_save
+from django.db import IntegrityError, models
+from django.db.models.signals import post_delete, post_save
+from django.dispatch import receiver
+from django.utils.translation import gettext_lazy as _
+from django_prometheus.models import ExportModelOperationsMixin
+from loguru import logger
+
+
+
+INSULT_REFERENCE_ID_PREFIX_OPTIONS: List[str] =  [
```

Ruff (0.12.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)